.github/workflows/link-checker.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         id: lychee
         uses: lycheeverse/lychee-action@v1
         with:
-          args: --verbose --max-retries 5 --retry-wait-time 10 --accept=200,403,429 --exclude-path tests/ ./ release-notes/
+          args: --verbose --max-retries 5 --retry-wait-time 10 --accept=200,403,429 --exclude-path tests/ --exclude-path release-notes/ ./
```

Comment:
Incorrect rebase?

Reply:
Seems like the syntax is not correct. Trying to fix it.